README.md

```diff
@@ -17,37 +17,53 @@ npm install @wert-io/module-react-component
 ## Usage example
 
 ```
-import WertModule from '@wert-io/module-react-component';
+import { useWertWidget } from '@wert-io/module-react-component';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      wertOptions: {
-        partner_id: '{{YOUR_PARTNER_ID}}',
+export default function WidgetButton () {
+    const options = {
+        partner_id: 'YOUR_PARTNER_ID',
         listeners: {
-          loaded: () => console.log('loaded'),
+            loaded: () => console.log('loaded'),
         },
+    };
 
-        ...
+    const { open: openWertWidget } = useWertWidget(options);
```

Comment:
Do you think this example is overall clear enough?
I've also shown how to rename the method because this is what everyone will probably have to do anyway (our method names are generic)